src/client/components/lazy-chat/lazy-chat.vue

```diff
@@ -74,4 +74,62 @@ export default {
 .lazy-chat__prompt {
   box-shadow: var(--shadow-wide-grey);
 }
+
+.lazy-chat__open-button {
+  width: 54px;
+  height: 54px;
+  border-radius: 50%;
+  padding: 0;
+  margin-right: -1px;
+  margin-bottom: -1px;
+  position: relative;
+}
+
+.lazy-chat__open-button:before {
+  content: '';
+  display: inline-block;
+  margin-left: 0;
+  margin-bottom: -6px;
+  width: 32px;
+  height: 26px;
+  background-image: url("/images/chat-bubble.svg");
+  background-position: center !important;
+  background-size: contain !important;
+  background-repeat: no-repeat !important;
+}
+
+.loading:before {
+  animation: button__spinner 2s linear infinite;
+  background-image: url('/images/spinner.png');
+  margin-bottom: 0 !important;
```

Comment:
You should have a really, really, really good reason to use `!important` :) If you positively absolutely need it, it should be documented. Otherwise, fix it (I can help you, or maybe Bas)